music21/midi/__init__.py

```diff
@@ -829,12 +829,20 @@ def parseChannelVoiceMessage(self, midiBytes: bytes) -> bytes:
                 self.parameter1 = byte1  # this is the controller id
                 self.parameter2 = byte2  # this is the controller value
             return midiBytes[3:]
-        else:
-            # NOTE_ON and NOTE_OFF
+        elif self.type == ChannelVoiceMessages.PITCH_BEND:
+            self.parameter1 = byte1  # least significant byte
+            self.parameter2 = byte2  # most significant byte
+            return midiBytes[3:]
+        elif self.type in (ChannelVoiceMessages.NOTE_ON, ChannelVoiceMessages.NOTE_OFF):
             # next two bytes:  pitch, velocity
             self.pitch = byte1
             self.velocity = byte2
             return midiBytes[3:]
+        elif self.type == ChannelVoiceMessages.POLYPHONIC_KEY_PRESSURE:
+            self.pitch = byte1
+            self.data = byte2
```

Comment:
Should this be parameter1, parameter2?

Reply:
The values are pitch and pressure. Up to you.

Reply:
The monophonic pressure message uses `self.data` for pressure, so that's what I was thinking.

Reply:
Looking more closely, .pitch and .data are both just synonyms for parameter 1 on setting; on getting, .pitch will only return the value if NOTE_ON or NOTE_OFF. so we definitely want parameter 1 and parameter2 because we'll lose byte2. Now that I see this I see that this patch is mostly just about readability of the code.